src/api/app/models/user.rb

```diff
@@ -63,6 +63,7 @@ class User < ApplicationRecord
   has_many :rss_feed_items, -> { order(created_at: :desc) }, class_name: 'Notification::RssFeedItem', as: :subscriber, dependent: :destroy
 
   scope :all_without_nobody, -> { where('login != ?', nobody_login) }
+  scope :autocomplete_login, ->(prefix) { prefix.present? ? where('login LIKE ?', "#{prefix}%") : User.none }
```

Comment:
Can you maybe change the name of the scope to something like "with_login_prefix" ?? that way when you use it it will be something like "User.with_login_prefix('foo')"